### ISSUE TITLE
ErrorException: unserialize(): Error at offset 0 of 40 bytes

### ISSUE THREAD

**Issue by marc-hanheide** [LCAS/pgr_system#50]
https://sentry.io/university-of-lincoln/pgr-system/issues/665451616/

```
ErrorException: unserialize(): Error at offset 0 of 40 bytes
  File "app/Http/Middleware/DevUser.php", line 25, in handle
    return $next($request);
  File "app/Http/Middleware/RoutePrefixProxy.php", line 25, in handle
    return $next($request);
  File "public/index.php", line 52
    $request = Illuminate\Http\Request::capture()
...
(33 additional frame(s) were not displayed)
```

**Comment by marc-hanheide** [comment]
@pet1330 the problem is that the cookies from https://lcas.lincoln.ac.uk/pgr-test are no conflicting with https://lcas.lincoln.ac.uk/pgr 

https://stackoverflow.com/questions/51817913/unserialize-error-at-offset-0-of-40-bytes-error

Clearing cookies helps. How can I ensure the cookies don't get mixed up?

**Comment by marc-hanheide** [comment]
See https://github.com/pet1330/pgr_system/blob/d3bc08964b119826bc75db074faae72de48c0747/config/session.php#L125 and make cookie domain dependent

**Comment by marc-hanheide** [comment]
fixed with #51
